src/coprocessor/dag/handler.rs

```diff
@@ -71,10 +71,24 @@ impl DAGRequestHandler {
             ranges,
             config.clone(),
         )?;
+
+        // Check output offsets
+        let output_offsets = req.take_output_offsets();
+        let schema_len = out_most_executor.schema().len();
+        for offset in &output_offsets {
+            if (*offset as usize) >= schema_len {
```

Comment:
Does any test cover it?

Reply:
no

Reply:
This pr does two different things. This is not recommended.

Reply:
Some call stack is missing in backtrace, removing these inline *may* help.